### ISSUE TITLE
Double the pan-speed on PC users

### ISSUE THREAD

**Issue by CoalRock8642** [Anuken/Mindustry-Suggestions#2813]
### Describe what you would like changed, and why.

I would like to pan faster and see things quick and at a glance.

### Describe the changes you want to propose. Include possible alternatives.

Double the panning speeds

**Comment by JrTRinny** [comment]
you can just `L-Shift + Pan View`

**Comment by CoalRock8642** [comment]
I said more max speed, even with **L-Shift + Pan View**

**Comment by the-kido** [comment]
What about holding middle-click + scrolling?

**Comment by CoalRock8642** [comment]
I need more max speed. I asked. Can you make it Double of current? No then make it no less then 1.4 times the current.
Please either make it. Or I can understand the "close issue" as a no.

**Comment by CoalRock8642** [comment]
If you need more things. You can increase the panning speed on "L-Shift + Pan View" That will also be fine. Please don't ask me controls. I only ask speeds.

**Comment by JrTRinny** [comment]
you know that `L-Shift + Pan View` is fast enough that speed is not a problem? who'd need that fast pan view speed except you

**Comment by Humanoid-X** [comment]
Who even need that fast.......................

**Comment by CoalRock8642** [comment]
My GOD. I asked, more pan speeds, no less than 1.4 times, double is what i ask for. You have problems?

Final time, I want increase panning speeds to double, if not not less than 1.4 times, both on shift and without shift.
You cannot do, just close the thread.

Good Day,
COALROCK8642

**Comment by JrTRinny** [comment]
yes, we know, you asked your *more panning speed*, and I asked, "*isn't `L-Shift + Pan View` not enough*", also, who'd need that much panning speed except you, what part did you not understand?

**Comment by CoalRock8642** [comment]
How about. left alt as 3rd button for speed. Like how foo's client did. or copy how much u can. more panning speed is alaways a plus.

**Comment by Humanoid-X** [comment]
The question is, why do u want it? It has no practical use case.

**Comment by OnyxFinger** [comment]
maybe it would be possible to use pan view on the minimap to insta-pan over there
it makes sense and pan viewing all the way is annoying even with speed, escpecially on big maps

**Comment by CoalRock8642** [comment]
Maps huge currents speed slow
